new-log-viewer/src/components/StatusBar/LogLevelSelect/index.tsx

```diff
@@ -0,0 +1,272 @@
+ * @param props.children
+ * @param props.rest
+ * @return
+ */
+const PlacementLeftTooltip = ({children, ...rest}: TooltipProps) => (
+    <Tooltip
+        {...rest}
+        placement={"left"}
+    >
+        {children}
+    </Tooltip>
+);
+
+/**
+ * Renders a `<Tooltip/>` with its placement set to the right of the children element.
+ *
+ * @param props
+ * @param props.children
+ * @param props.rest
+ * @return
+ */
+const PlacementRightTooltip = ({children, ...rest}: TooltipProps) => (
+    <Tooltip
+        {...rest}
+        placement={"right"}
+    >
+        {children}
+    </Tooltip>
+);
+
+interface LogSelectOptionProps {
+    isChecked: boolean,
+    logLevelName: string,
+    logLevelValue: LOG_LEVEL,
+    onCheckboxClick: React.MouseEventHandler
+}
+
+/**
+ * Renders an <Option/> in the <LogLevelSelect/> for selecting some log level and the level above
+ * it.
+ *
+ * @param props
+ * @param props.isChecked
+ * @param props.logLevelName
+ * @param props.logLevelValue
+ * @param props.onCheckboxClick
+ * @return
+ */
+const LogSelectOption = ({
+    isChecked,
+    logLevelName,
+    logLevelValue,
+    onCheckboxClick,
+}:LogSelectOptionProps) => {
+    return (
+        <Option
+            data-value={logLevelValue}
+            key={logLevelName}
+            value={logLevelValue}
+        >
+            <ListItemDecorator>
+                <PlacementLeftTooltip
+                    title={
+                        <Stack
+                            alignItems={"center"}
+                            direction={"row"}
+                        >
+                            {isChecked ?
+                                <RemoveIcon/> :
+                                <AddIcon/>}
+                            {logLevelName}
+                        </Stack>
+                    }
+                >
+                    <Checkbox
+                        checked={isChecked}
+                        size={"sm"}
+                        value={logLevelValue}
+                        onClick={onCheckboxClick}/>
+                </PlacementLeftTooltip>
+            </ListItemDecorator>
+            <PlacementRightTooltip
+                title={
+                    <Stack
+                        alignItems={"center"}
+                        direction={"row"}
+                    >
+                        {logLevelName}
+                        {" and below"}
+                    </Stack>
+                }
+            >
+                <ListItemContent data-value={logLevelValue}>
+                    {logLevelName}
+                </ListItemContent>
+            </PlacementRightTooltip>
+        </Option>
+    );
+};
+
+interface ClearFiltersOptionProps {
+    onClick: () => void
+}
+
+/**
+ * Renders an <Option/> to clear all filters in the <LogLevelSelect/>.
+ *
+ * @param props
+ * @param props.onClick
+ * @return
+ */
+const ClearFiltersOption = ({onClick}: ClearFiltersOptionProps) => (
+    <Option
+        value={-1}
```

Comment:
Shall we create something like `const INVALID_LOG_LEVEL_SELECT_OPTION_VALUE = -1;`?

* Or instead of using -1, would be make sense to use `LOG_LEVEL.UNKNOWN - 1`?
* I haven't tried but `null` might also work here.